components/Post.tsx

```diff
@@ -149,7 +152,9 @@ export const Post = ({ cmsData }: PostProps) => {
 
                 {memberSubscriptions && <Subscribe {...{ settings }} />}
 
-                {commento.enable && <Comments {...{ id: post.id, url: commento.url }} />}
+                {disqus.enable && (<DisqusComments {...{ post: post, shortname: disqus.Shortname }} />)}
```

Comment:
Maybe only pass `post.id` and `post.url` instead of the whole post object?

Reply:
Question: What does disqus need the `post.url` for, shouldn't `post.id` be sufficient just like in commento?

Reply:
It's the way Disqus works. Conversations are linked to the Id and it uses the URL to discover/suggest other conversations on your site. Personally, I think that 3 parameters is limit as for the readability, but I don't mind changing it if you prefer. It just seemed easier at the time as all those properties were available on the PageOrPost type